src/openforms/submissions/api/serializers.py

```diff
@@ -292,6 +292,8 @@ class Meta:
     def update(self, instance, validated_data):
         email = validated_data.pop("email")
         instance.suspended_on = timezone.now()
+        if hasattr(instance, "auth_info"):
+            instance.auth_info.hash_identifying_attributes()
```

Comment:
```suggestion
        if instance.is_authenticated:
            instance.auth_info.hash_identifying_attributes()
```
effectively the same, but this conveys the message better